src/components/forms/ReviewSMSTemplate/ReviewSMSTemplate.tsx

```diff
@@ -3,40 +3,50 @@
 import { PreviewTemplate } from '@molecules/PreviewTemplate';
 import { ReviewTemplate } from '@organisms/ReviewTemplate';
 import content from '@content/content';
-import { renderMarkdown } from './server-actions';
-
-export type ReviewSMSTemplateProps = {
-  templateName: string;
-  message: string;
-};
+import { PageComponentProps } from '@utils/types';
+import { useFormState } from 'react-dom';
+import Link from 'next/link';
+import { ChevronLeftIcon } from 'nhsuk-react-components';
+import { renderMarkdown, reviewSmsTemplateAction } from './server-actions';
 
 export function ReviewSMSTemplate({
-  templateName,
-  message,
-}: Readonly<ReviewSMSTemplateProps>) {
+  initialState,
```

Comment:
Should the name of this form be `PreviewSMSTemplate` since this form will only be used by the the preview-text-message-template page?

Reply:
Can do? The NHSApp component is named Similar. 

Might be one we could bundle into your PR? So they're all consistent?